src/templates/menu.html

```diff
@@ -16,6 +16,7 @@
 			<div class="footer">
 				<a class="pull-left icon icon-home" href="http://cletusc.github.io/Userscript--Twitch-Chat-Emotes" target="_blank" title="Visit the homepage where you can donate, post a review, or contact the developer"></a>
 				<a class="pull-left icon icon-gear" data-command="toggle-editing" title="Toggle edit mode"></a>
+				<input type="text" placeholder="Search emotes" data-command="search-emotes">
```

Comment:
i somehow missed this pr. this probably needs some styling to make it fit in better to the emote menu. can you propose some?

Reply:
Added some simple styling that should be fitting to the overall look.